components/TodoList.tsx

```diff
@@ -1,3 +1,35 @@
-export function TodoList() {
-  return <></>;
+import { Text, ListItem, Stack } from "tamagui";
+
+import { TodoContext, TodoStore } from "../stores/Todo.store";
+import { observer } from "mobx-react-lite";
+import { useContext } from "react";
+
+function EmptyTodo() {
+  return (
+    <Text>No to-dos created... make one with the button in the corner!</Text>
+  );
 }
+
+const ListTodos = observer(function ListTodos() {
+  const store = useContext(TodoContext)!;
+
+  return store.todos.map((todo) => <ListItem title={todo.text} />);
+});
+
+const TodoContents = observer(function TodoContents() {
+  const store = useContext(TodoContext)!;
+
+  if (store.todos.length === 0) {
+    return <EmptyTodo />;
+  } else {
+    return <ListTodos />;
+  }
+});
```

Comment:
```suggestion
return store.todos.length ? <ListTodos/> : <EmptyTodo/>;
```

Reply:
I feel like it reads better personally as is, and using a ternary seems like prioritizing line count over readability.